src/specifications/profile-resolution/profile-resolution-specml.xml

```diff
@@ -3,1772 +3,1370 @@
 
+      <p>An external reference appears as an absolute or relative URL:</p>
+      <tagging whose="source_profile"><![CDATA[
+profile:
+  uuid: ~~~
+  metadata: ~~~
+  imports:
+    - href: >-
+        https://github.com/usnistgov/oscal-content/tree
+        /master/nist.gov/SP800-53/rev4/xml/NIST_SP-800-53_rev4_catalog.xml
+      include-controls: ${{ list of selected controls }}
+    - href: "../../NIST_SP-800-53_rev5_catalog.xml"
+      include-controls: ${{ list of selected controls }} ]]>
+      </tagging>
+      <p>All import directives will contain either <src>include-all: ~</src> or
+          <src>include-controls</src>. These directives indicate which controls from the imported
+        document are explicitly selected <xref rid="select-phase"/>.</p>
+      <p>The following section contains requirements for processing the "import" child of a source
+        profile.</p>
+      <section id="importhrefs">
+        <head>Import href Requirements</head>
+
+        <section id="resolving-uris">
+          <head>Import URI Resolution</head>
+          <p>Tools MUST resolve URIs by following <a
+              href="https://datatracker.ietf.org/doc/html/rfc3986#section-5">Section 5 of
+              RFC3986</a>, with the exception of URI Fragments (URIs that start with "#"). URI
+            Fragments MUST instead be resolved as defined in <xref rid="internalref"/>.</p>
+        </section>
+
+        <section id="aquiring-resource">
+          <head>Import Resource Acquisition</head>
+          <p>Tools MUST acquire resources at the resolved URI by following <a
+              href="https://datatracker.ietf.org/doc/html/rfc3986#section-5">Section 5 of
+              RFC3986</a>, with the exception of URI Fragments (URIs that start with "#"). URI
+            Fragments MUST instead be acquired as defined in <xref rid="internalref"/>.</p>
+          <p>In the case that acquiring a resource fails, the tool MUST cease processing and provide
+            an error. In order to ensure profile resolution results in the same Catalog regardless
+            of which tool resolves it, all imports must successfully resolve. While this may cause
+            inconvenience if resources are frequently not available, it ensures
+            interoperability.</p>
+        </section>
+        <section id="internalref">
+          <head>Internal References</head>
+          <p>URI Fragments in OSCAL represent internal references to other OSCAL objects in the same
+            document. These references follow the pattern of "#{{objectID}}". For example, the URI
+            Fragment "#param1" is referencing the Parameter with unique ID "param1".</p>
+          <p>Tools MUST resolve and acquire URI Fragments by fetching the OSCAL object referenced.
+            The reference must only be searched for within the list of IDs of all objects in the
+            document in which the reference appears.</p>
```

Comment:
```suggestion
            The referenced identifier MUST only be searched for within the scope of IDs for which the reference type allows.</p>
```
Specific type of references have unique ID scopes. We will need to get into these in this spec at some point.